Source/Scene/Expression.js

```diff
@@ -263,10 +319,7 @@ define([
             op = ast.operator;
             left = createRuntimeAst(expression, ast.left);
             right = createRuntimeAst(expression, ast.right);
-            if (op === '+' || op === '-' || op === '*' ||
-                op === '/' || op === '%' || op === '===' ||
-                op === '!==' || op === '>' || op === '>=' ||
-                op === '<' || op === '<=') {
+            if (binaryOperators.indexOf(op) > -1) {
```

Comment:
This may have hurt "compiling" performance, which will be important for the responsive of a UI generating styles.  I don't know for sure so we can see what a profile looks like later; at the least, we can order the linear search so the most common operators are first.